ui/app/serializers/pki/tidy.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import ApplicationSerializer from '../application';
+
+export default class PkiIssuerSerializer extends ApplicationSerializer {
```

Comment:
Could you update this to be `PkiTidySerializer`?

Reply:
haha of course! copy pasta 🍝